src/main/java/pl/javastart/task/FormaOpodatkowania.java

```diff
@@ -2,8 +2,11 @@
 
 public class FormaOpodatkowania {
 
+    public static final double KWOTA_WOLNA_OD_PODATKU = 10000.0;
+
     public double wyliczPodatek(double przychody) {
```

Comment:
Tutaj powinieneś mieć dodatkowy parametr z wydatkami, żeby nie traktować parametru raz jako przychód, a raz jako dochód. Tam, gdzie nie będzie to potrzebne, parametr z wydatkami będziesz mógł po prostu zignorować, ale kontrakt zawsze będzie taki sam